table/writer.go

```diff
@@ -33,6 +33,7 @@ type Writer interface {
 	SortBy(sortBy []SortBy)
 	Style() *Style
 	SuppressEmptyColumns()
+	SuppressLastColumnSpaces()
```

Comment:
`SupressTrailingSpaces` may be a better name for this since this method is at a Table level.